lib/prompts/expand.js

```diff
@@ -218,13 +218,18 @@ Prompt.prototype.validateChoices = function (choices) {
 /**
  * Generate a string out of the choices keys
  * @param  {Array}  choices
- * @param  {Number} defaultIndex - the choice index to capitalize
+ * @param  {Number|String} default - the choice index or name to capitalize
  * @return {String} The rendered choices key string
  */
-Prompt.prototype.generateChoicesString = function (choices, defaultIndex) {
+Prompt.prototype.generateChoicesString = function (choices, defaultChoice) {
   var defIndex = choices.realLength - 1;
-  if (_.isNumber(defaultIndex) && this.opt.choices.getChoice(defaultIndex)) {
-    defIndex = defaultIndex;
+  if (_.isNumber(defaultChoice) && this.opt.choices.getChoice(defaultChoice)) {
+    defIndex = defaultChoice;
+  } else if (_.isString(defaultChoice)) {
+    let index = _.flatMap(choices.realChoices, function(c) {
+      return c.value;
+    }).indexOf(defaultChoice);
+    defIndex = (index !== -1 ? index : index); 
```

Comment:
Overall looks good, just same comment as the other review about this piece `(index !== -1 ? index : index);`. It's cleaner (less code branches) if we use `Math.max(index, 0)`.

Reply:
I realize now that this was a bug, it should be `defIndex = (index !== -1 ? defIndex : index);`

I'll make the change in the commit to use `_.findIndex`, though I don't think we can use Math.max anymore with that, since expand's default is the end of the array.